### PR TITLE
Updated: Provide Service Account as JSON for GCSBucketCreate

### PR DESCRIPTION
**Pull request by ardian4**
Updated: Provide Service Account as JSON for GCS Bucket Create plugin

Jira Ticket: https://issues.cask.co/browse/PLUGIN-170

### CODE REVIEW

File: src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketCreate.java [hunk 1]
```diff
@@ -67,13 +68,23 @@ public void run(ActionContext context) throws Exception {
...
-    String serviceAccountFilePath = config.getServiceAccountFilePath();
-    Credentials credentials = serviceAccountFilePath == null ?
-                                null : GCPUtils.loadServiceAccountCredentials(serviceAccountFilePath);
-
-    if (serviceAccountFilePath != null) {
-      configuration.set("google.cloud.auth.service.account.json.keyfile", serviceAccountFilePath);
+    Boolean isServiceAccountFilePath = config.isServiceAccountFilePath();
+    if (isServiceAccountFilePath == null) {
+      context.getFailureCollector().addFailure("Service account type is undefined.",
+                                               "Must be `File Path` or `JSON`");
```
Comment (CuriousVini) [inline src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketCreate.java]: `It must be `filePath` or `JSON``

File: src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketCreate.java [hunk 2]
```diff
@@ -67,13 +68,23 @@ public void run(ActionContext context) throws Exception {
...
     Configuration configuration = new Configuration();
 
-    String serviceAccountFilePath = config.getServiceAccountFilePath();
-    Credentials credentials = serviceAccountFilePath == null ?
-                                null : GCPUtils.loadServiceAccountCredentials(serviceAccountFilePath);
-
-    if (serviceAccountFilePath != null) {
-      configuration.set("google.cloud.auth.service.account.json.keyfile", serviceAccountFilePath);
+    Boolean isServiceAccountFilePath = config.isServiceAccountFilePath();
+    if (isServiceAccountFilePath == null) {
```
Comment (CuriousVini) [inline src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketCreate.java:72]: check for null and empty

Comment (ardian4) [reply src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketCreate.java:72]: @CuriousVini `isServiceAccountFilePath` is `Boolean` type.  Am I missing something ?

**Review by CuriousVini — APPROVED** [review]
lgtm 👍 
Please squash commits

**Review by CuriousVini — APPROVED** [review]

**Pull request merged**